tests/e2e/pages/breachPage.js

```diff
@@ -0,0 +1,29 @@
+exports.BreachPage = class BreachPage {
+  constructor(page) {
+    this.page = page;
+    this.loginButton = page.locator('#sign-in-btn');
+    this.vpnBanner = page.locator('.vpn-banner');
+    this.homePageNavButton = page.locator(
+      '.desktop-menu [data-event-label="Home"]'
+    );
+    this.SecTipsNavButton = page.locator(
```

Comment:
nit: Another instance of a locator being pascal-vs-camel case. :shrug:

And just because I can't resist a good <kbd>git grep</kbd>&hellip;
```sh
git grep -En "\bthis\.[A-Z][a-zA-Z]+\b" | cat

tests/e2e/pages/breachPage.js:9:    this.SecTipsNavButton = page.locator(
tests/e2e/pages/securityTipsPage.js:19:    await this.SecurityPageUI.elementHandle();
tests/e2e/pages/securityTipsPage.js:23:    return this.BreachCards;
```